### ISSUE TITLE
Add `aria-brailleroledescription` to B display

### ISSUE THREAD

**Issue by jooyoungseo** [xability/maidr-legacy#239]
When B mode is activated in text input  area, the first two columns are used to indicate the field type in braille (i.e., ed for edit field). We need to save this space for braille. This is possible if you assign `aria-brailleroledescription = ""`. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-brailleroledescription

**Comment by ellvix** [comment]
Oh cool! I wonder if `role="application"` would do the same thing.

Will this still say "editable" when focused by JAWS or NVDA?

**Comment by ellvix** [comment]
This may not work as we want. aria-brailleroledescription cannot be blank / empty, and must be different than aria-roledescription which is also different than default role. We can set all this, but we still need a single character aria-brailleroledescription.
